src/interfaces/ITolocars.ts

```diff
@@ -0,0 +1,11 @@
+export interface ITolocarsFrontmatter {
+  title: string;
+  subtitle?: string;
+  date?: string;
+  footerGrey?: boolean;
```

Comment:
In the Layout files, you're hardcoding `footerGrey` to be true. So we can't change it via frontmatter anymore. I would just remove it from this type.